llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -2534,6 +2534,8 @@ bool SPIRVInstructionSelector::selectIntrinsic(Register ResVReg,
   }
   case Intrinsic::spv_step:
     return selectExtInst(ResVReg, ResType, I, CL::step, GL::Step);
+  case Intrinsic::spv_radians:
+    return selectExtInst(ResVReg, ResType, I, CL::radians, GL::Radians);
```

Comment:
Do we need a test for CL::radians?

Reply:
Modified the test to also test the CL variant.

Reply:
We don't use the opencl extension in hlsl though. Even though this is an easy test change `llvm/test/CodeGen/SPIRV/hlsl-intrinsics` tests should be devoted to how HLSL does things.

Reply:
CL variant could support double type and vector2/3/4/8/16.
It would be nice to have these tested.

Reply:
> We don't use the opencl extension in hlsl though. Even though this is an easy test change `llvm/test/CodeGen/SPIRV/hlsl-intrinsics` tests should be devoted to how HLSL does things. I feel like the folks that care about opencl should be adding opencl tests to their parts of the test directory.

Can we have a common place to test things like int_spv_radians so that we can share the same test with different run lines, similar to what Adam did in [radians.ll](https://github.com/llvm/llvm-project/pull/110800/files#diff-291ac3597dffe1aaedd74f8452fd868bcaf8dc572efc6114db2020fb5187faa5)?

Reply:
I’ll let other folks weigh in. I was afraid of testing scope creep. We do want a set of tests that are hlsl only.  The question is do we want to be on the hook for opencl tests and do we want to go and update every hlsl intrinsic to have cl tests as well?

Reply:
We are adding the `spv_radians` -> `cl::radians` lowering code path, so we should test it.

Reply:
Thats because `selectExtInst` defined in such a way where GL is optional but CL is required. Also I'm not against testing `cl::radians`.  I just don't think it should be a part of the `llvm/test/CodeGen/SPIRV/hlsl-intrinsics` test suite.

Reply:
I separated the cl specific tests to [llvm/test/CodeGen/SPIRV/opencl/radians.ll](https://github.com/llvm/llvm-project/pull/110800/files#diff-475b96ef54c892f11b256ed84dc5169abcebe11ae128fac4bebb402fac7e0bbe)

Reply:
>I separated the cl specific tests to llvm/test/CodeGen/SPIRV/opencl/radians.ll

@adam-yang Thank you!